renderer/components/ManualInputForm.tsx

```diff
@@ -61,11 +64,29 @@ export default function ManualInputForm({
     setReport(report);
   };
 
-  const copyLineHandler = (event) => {
-    if ((event.ctrlKey || event.metaKey) && event.code === "KeyD") {
-      event.preventDefault();
+  const textAreaKeyHandler = (e) => {
+    if ((e.ctrlKey || e.metaKey) && e.code === "KeyD") {
+      e.preventDefault();
```

Comment:
If I'm not mistaken, To avoid duplicating code you can move preventDefault from each condition and write it above them

Reply:
I've checked. I'm mistaken with this comment.